hack/istio/functions.sh

```diff
@@ -30,7 +31,7 @@ EOM
     ${CLIENT_EXE} wait --for condition=Ready -n ${ISTIO_NAMESPACE} smmr/default --timeout 300s
   fi
 
-  if [ "${ENABLE_INJECTION}" == "true" ]; then
+  if [ "${ENABLE_INJECTION}" == "true" ] || [ "${AUTO_INJECTION}" == "true" ]; then
```

Comment:
This seems relevant even for non-Maistra environments (I don't see anything here that restricts this to only maistra environments)

Reply:
the function here is named `prepare_maistra` , so I expected that it is related only to `maistra`. Also, this function is not in the main branch anymore.